File: maths101/maths.js
var goat_count = 5;

var add_goats = function (new_goats) { 
    goat_count += new_goats;
};

add_goats(2);
add_goats(7);
add_goats(12);
add_goats(54);
add_goats(34);
add_goats(215);

console.log("We have " + goat_count + " js goats");

var sum = function (a, b) {
    return a + b;
};

var orange_count = 54;
var banana_count = 21;
var total_fruits = sum(
    orange_count,
    banana_count
);

console.log(
    "We have " 
    + orange_count + " js oranges, "
    + banana_count + " js bananas, and "
    + total_fruits + " js total fruits"
)

var goose_count = 74
var blue_bird_count = 631
var total_birds = sum(goose_count, blue_bird_count)




console.log(
    "We have " 
    + goose_count + " js gooses, "
    + blue_bird_count + " js bluebirds, and "
    + total_birds + " js total bird"
)

File: maths101/maths.py
goat_count = 5

def add_some_goats(number_of_goats_to_add):
    global goat_count
    goat_count += number_of_goats_to_add

add_some_goats(2)
add_some_goats(7)
add_some_goats(12)
add_some_goats(54)
add_some_goats(34)
add_some_goats(25)

print("We have " + str(goat_count) + " py goats")

def add_two_numbers_and_give_me_the_result(a, b):
    return a + b


orange_count = 54
banana_count = 21
total_fruits = add_two_numbers_and_give_me_the_result(
    orange_count,
    banana_count
)

print(
    "We have " 
    + str(orange_count) + " py oranges, "
    + str(banana_count) + " py bananas, and "
    + str(total_fruits) + " py total fruits"
)

goose_count = 74
blue_bird_count = 631
total_birds = add_two_numbers_and_give_me_the_result(
    goose_count,
    blue_bird_count
)

print(
    "We have " 
    + str(goose_count) + " py gooses, "
    + str(blue_bird_count) + " py bluebirds, and "
    + str(total_birds) + " py total bird"
)

